conf/cassandra.yaml

```diff
@@ -2076,6 +2076,15 @@ drop_compact_storage_enabled: false
 # if zero_ttl_on_twcs_enabled is set to false, this property is irrelevant as such statements will fail.
 #zero_ttl_on_twcs_warned: true
 
+# Guardrail enabling querying a table with indexes without specifying partition keys, defaults to true
+#non_partition_key_restricted_query_enabled: true
+# Maximum number of referenced indexes on a replica when executing a SELECT query without partition key restrictions
+# before emitting a warning, defaults to 32.
+#maximum_sstables_with_non_paritition_key_restricted_query_warn_threshold: 32
```

Comment:
How about `sai_sstable_indexes_per_query_warn_threshold` and `sai_sstable_indexes_per_query_fail_threshold `? We communicate this is an SAI-only thing, make it clear that it's per query, etc.